go/vt/vttablet/tabletserver/schema/db.go

```diff
@@ -372,3 +390,72 @@ func reloadDataInDB(ctx context.Context, conn *connpool.DBConn, altered []*Table
 	}
 	return nil
 }
+
+// GetFetchViewQuery gets the fetch query to run for getting the listed views. If no views are provided, then all the views are fetched.
+func GetFetchViewQuery(viewNames []string) (string, error) {
```

Comment:
all 3 methods look identical, may be they can be placed in a common function.

Reply:
I could do that, but then they'll have a bunch of switch cases, which isn't great either.